JavaScript/JavaScriptSDK/Util.ts

```diff
@@ -453,7 +453,7 @@ module Microsoft.ApplicationInsights {
 
         public static parseUrl(url): HTMLAnchorElement {
             if (!UrlHelper.htmlAnchorElement) {
-                UrlHelper.htmlAnchorElement = UrlHelper.document.createElement('a');
+                UrlHelper.htmlAnchorElement = !!UrlHelper.document.createElement ? UrlHelper.document.createElement('a'): {};
```

Comment:
We rely on ability to create <a> to parse url. With this fix we're fixing the exception but as far as I understand we won't be able to parse URL, which means the end-to-end scenario is still broken.
Can you describe how you reproduced it? Did you run the sdk in browser?